pkg/pxc/service.go

```diff
@@ -0,0 +1,87 @@
+package pxc
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	api "github.com/Percona-Lab/percona-xtradb-cluster-operator/pkg/apis/pxc/v1alpha1"
+)
+
+func (h *PXC) newServiceNodes(cr *api.PerconaXtraDBCluster) *corev1.Service {
+	obj := &corev1.Service{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: "v1",
+			Kind:       "Service",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      cr.Name + "-pxc-nodes",
+			Namespace: cr.Namespace,
+			Annotations: map[string]string{
+				"service.alpha.kubernetes.io/tolerate-unready-endpoints": "true",
+			},
+			Labels: map[string]string{
+				"app":     "pxc",
```

Comment:
not sure, make a constant here can have sense, up to you

Reply:
done